Css frameworks

Christelmarita opened the pull request.

Christelmarita committed:
all finished pages added

Christelmarita committed:
titles and indentation changed

Christelmarita committed:
footer added and some col changed

sulenchy commented on .gitignore:
```diff
@@ -1,3 +1 @@
 node_modules/
-.DS_Store
-dist/
```
You don't need to remove this

sulenchy reviewed (approved):
LGTM

Christelmarita committed:
Update .gitignore

Added back .DS_Store and dist/ which I had previously removed, after getting feedback from teacher

Christelmarita committed:
changed logo and removed inline styling from logo. Changed some margins.

Christelmarita committed:
Update README.md
